client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -237,25 +237,25 @@ export const PrescriptionLineEditForm: React.FC<
 
   const key = item?.id ?? 'new';
 
-  // //MOCK DATA FOR TESTING ABBREVIATIONS
-
-  const defaultDirections = [
-    { id: '1', item_link_id: 'a', direction: '1 per day', priority: 2 },
-    { id: '2', item_link_id: 'b', direction: '2 per day', priority: 3 },
-    { id: '3', item_link_id: 'c', direction: '3 per day', priority: 1 },
-  ];
-
-  // //END OF MOCK DATA
-
   const { data: options = [] } = useAbbreviations();
 
   const saveAbbreviation = () => {
     if (!abbreviation) return;
-
     const note = getPrescriptionDirections(abbreviation, options);
     updateNotes(note);
+    setDefaultDirection('');
   };
 
+  const saveDefaultDirection = useCallback(
+    (direction: string) => {
+      setDefaultDirection(direction);
+      const note = getPrescriptionDirections(direction, options);
+      updateNotes(note);
+      setAbbreviation('');
```

Comment:
When you set the directions, it could put into the abbreviation input similar to OG, rather than clearing the abbreviation input. Not a biggy though in the spirit of getting this merged!